x-pack/test/apm_api_integration/tests/transactions/transactions_groups_overview.ts

```diff
@@ -86,187 +71,48 @@ export default function ApiTest({ getService }: FtrProviderContext) {
             "APIRestController#order",
             "APIRestController#stats",
             "APIRestController#customerWhoBought",
+            "APIRestController#customer",
+            "APIRestController#topProducts",
+            "APIRestController#orders",
+            "APIRestController#product",
+            "ResourceHttpRequestHandler",
+            "APIRestController#products",
+            "DispatcherServlet#doPost",
           ]
         `);
 
-        expectSnapshot(response.body.transactionGroups.map((group: any) => group.impact))
-          .toMatchInline(`
+        const impacts = response.body.transactionGroups.map((group: any) => group.impact);
+        expectSnapshot(impacts).toMatchInline(`
           Array [
-            100,
-            1.43059146953109,
-            0.953769516915408,
-            0.905498741191481,
-            0.894989230293471,
+            93.9295870910491,
+            1.35334507158962,
+            0.905514602241759,
+            0.860178761411346,
+            0.850308244392878,
+            0.699947181217412,
+            0.476138685202191,
+            0.446650726277923,
+            0.262571482598846,
+            0.143906183235671,
+            0.062116281544223,
+            0.00973568923904662,
           ]
         `);
 
+        expect(sum(impacts)).to.eql(100);
```

Comment:
might be good to apply some sort of rounding here to prevent floating point issues?